_minutes/2023-09-11-wecg-tpac.md

```diff
@@ -0,0 +1,188 @@
+# WECG TPAC 2023, Public Notes—Sep 11, 2023
+
+ * Chair: Simeon Vincent
+ * Scribes: Devlin Cronin, Rob Wu
+ * Location: Seville, Spain
+ * Time: Mon 11 sep, 5pm - 6:30 PM CET (https://everytimezone.com/?t=64fe5880,384)
+ * Call-in details: [WebExtensions CG, 11th September 2023](https://www.w3.org/events/meetings/a884481a-dd80-45fe-a2dd-a90197791484/)
+
+
+## Agenda: [TPAC 2023](https://github.com/w3c/webextensions/issues/385)
+
+| Start | Duration | Description |
+| - | - | - |
+| 17:00 | 5 min | Intro |
+| 17:05 | 20 min | WECG meeting structure and follow-up |
+| 17:25 | 20 min | Automated compatibility testing |
+| 17:45 | 10 min | Break |
+| 17:55 | 20 min | Web API integration |
+| 18:15 | 15 min | Open discussion |
+| 18:30 | --- | End |
+
+
+## Attendees (sign yourself in)
+
+ 1. Oliver Dunk (Google Chrome)
+ 2. Devlin Cronin (Google Chrome)
+ 3. Rob Wu (Mozilla)
+ 4. Sam Macbeth (DuckDuckGo)
+ 5. Humero Noor (eyeo)
+ 6. Javier Fernand (Igalia)
+ 7. Kiara Rose (Apple)
+ 8. Simeon Vincent (Unaffiliated)
+ 9. Carlos Jeurissen (Jeurissen Apps)
+ 10. Tomislav Jovanovic (Mozilla)
+ 11. Maxim Tsoy (DuckDuckGo)
+ 12. Richard Worth (Capital One)
+ 13. Andrea (Igalia)
+ 14. Mukul Purohit (Microsoft)
+ 15. Patrick Kettner (Google Chrome)
+ 16. David Johnson (Apple)
+
```

Comment:
Appologies, I forgot to sign in. Can you add "17. Timothy Hatcher (Apple)"?

Thanks!